test/server/lib/validation/form_validators/ManageApplications.test.js

```diff
@@ -45,7 +45,7 @@ describe('server/lib/validation/form_validators/ManageApplication', () => {
     const stubValidDesc = sinon.stub(Validator.prototype, 'isValidDescription').returns(true);
     const stubValidUrl = sinon.stub(Validator.prototype, 'isValidUrl').returns(true);
     const stubKeys = sinon.stub(ManageApplication.prototype, '_formatIncomingPayload').returns(routeData.addApplication);
-    expect(ManageApplication.prototype.validateApplication(payload, false)).to.eventually.equal(true);
+    expect(ManageApplication.prototype.addApplication(payload)).to.eventually.equal(true);
```

Comment:
Thinking about this I wonder if we should have moved most of these tests to update and then stubbed that and just kept the env tests on add.

Reply:
We could have, it's a slight adjustment I'm happy to leave as it is.